miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/JsonArrayCollector.java

```diff
@@ -0,0 +1,46 @@
+package uk.ac.bbsrc.tgac.miso.webapp.util;
+
+import java.util.Collections;
+import java.util.Set;
+import java.util.function.BiConsumer;
+import java.util.function.BinaryOperator;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.stream.Collector;
+
+public class JsonArrayCollector implements Collector<java.lang.String, StringBuilder, java.lang.String> {
+
+  @Override
+  public BiConsumer<StringBuilder, String> accumulator() {
+    return (sb, metrics) -> {
+      if (sb.length() == 0) {
+        sb.append(metrics);
+      } else {
+        sb.append(".concat(");
```

Comment:
This is never getting stored in the DB. It's just being composited for the front end. We could parse it, but there's no good reason to bother since it's all being composited on the page.